data/the emerald sword.txt

```diff
@@ -0,0 +1,803 @@
+			`	"Well, he's paying me to help with this, so no problem with me. It's not like everyone gets to see the exotic things he finds first hand."`
+			`	John proceeds to examine the cube, picking it up and looking at it from different angles, trying to find some port to connect to. He finds one, but it's so deteriorated that any attempts to use it would be fruitless. He places the cube inside a machine to take a deeper scan of the object, and watches it silently working, staring patiently at the screen.`
+			`	"Yes, this seems to be a data storage device of some sort... it's made to be sturdy, like a black box, so it's plausible that we could recover some data from it. This is going to take a very long time, though."`
+			`	"How long?" You ask.`
+			`	"A couple months, probably. It depends on how alien this thing is, and if it's encrypted or not, but I should at least figure out if there is any recoverable data in here."`
+			`	John says he'll drop you a message once his results come out, and you part ways and return to your ship.`
+
+mission "The Box 3"
+	landing
+	name "The box"
+	description "Discover the whereabouts of the box from an information broker in the pirate world of Stormhold."
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Independent" "Neutral"
+		near "Sol" 100
+		attributes spaceport
+	destination "Stormhold"
+	
+	to offer
+		has "event: cube stolen"
+	
+	on offer
+		conversation
+			`You receive a call from John, who has been working on the mysterious Sheragi black box for little over a month now.`
+			`	"Captain <last>! I have some bad news and some worse news. which one do you want to hear first?"`
+			choice
+				`	"The bad news."`
+					goto bad
+				`	"The worse news."`
+					goto worse
+			label bad
+			`	"The bad news is, this is a lot more complicated than I thought. I found out that the data in the box has survived, and can still be recovered after all this time, but it will need some specialized hardware and knowledge, and we don't have that yet."`
+			`	"And the worse news?" You ask.`
+			`	Well... Some people invaded my lab during the night, and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone.`
+				goto whatnow
+			label worse
+			`	"Well... Some people invaded my lab during the night, and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone.`
+			`	"And the bad news?"`
+			`	"The bad news is, this is a lot more complicated than I thought. I found out that the data in the box has survived, and can still be recovered after all this time, but it will need some specialized hardware and knowledge, and we don't have that yet."`
+			label whatnow
+			`	"What do we do now? Do you have any idea where to find the box? Did you contact law enforcement?"
```

Comment:
Needs a backtick at the end of the line here.

I pushed a PR to your master with the missing backtick for this and the one in line 439, just to make an already easy task even easier.